### ISSUE TITLE
Class 'TYPO3\CMS\Core\Configuration\ExtensionConfiguration' not found

### ISSUE THREAD

**Issue by FireLizard** [tommysack/typo3-restclient#9]
Hi,

when upgrading to v2.0.4 the following error appears on TYPO3 7.6.32
```
Core: Exception handler (WEB): Uncaught TYPO3 Exception: Class 'TYPO3\CMS\Core\Configuration\ExtensionConfiguration' not found | Error thrown in file /var/www/html/dhmd.de/htdocs/releases/current/vendor/typo3/cms/typo3/sysext/core/Classes/Utility/GeneralUtility.php in line 4566
```

Can you help, please?

**Comment by tommysack** [comment]
Hy Sebastian, at the moment you can sanize the code replacing
HttpClient.php line code:

$extConf = GeneralUtility::makeInstance(ExtensionConfiguration::class) ->
get('restclient');

with

$extConf =
unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['restclient'])

I will release a patch as soon as I can.

Thank you


Il giorno lun 11 mar 2019 alle ore 14:56 Sebastian Gierth <
notifications@github.com> ha scritto:

**Comment by tommysack** [comment]
> Hi,
> 
> when upgrading to v2.0.4 the following error appears on TYPO3 7.6.32
> 
> ```
> Core: Exception handler (WEB): Uncaught TYPO3 Exception: Class 'TYPO3\CMS\Core\Configuration\ExtensionConfiguration' not found | Error thrown in file /var/www/html/dhmd.de/htdocs/releases/current/vendor/typo3/cms/typo3/sysext/core/Classes/Utility/GeneralUtility.php in line 4566
> ```
> Can you help, please?

Hy Sebastian, i have released the 2.0.5 that resolve the regression.
Bye
